app/views/requests/received_requests/_received_request_card.html.erb

```diff
@@ -7,21 +7,13 @@
     </p>
     <div class="card-left-bottom-row">
       <div class="card-left-bottom-button-space"></div>
-      <div class="card-left-bottom-button btn btn-success btn-block">
-        Accept
-      </div>
+      <%= link_to 'Accept', accept_request_path(request), method: :patch, class: "card-left-bottom-button btn btn-success btn-block" %>
       <div class="card-left-bottom-button-space"></div>
-      <div class="card-left-bottom-button btn btn-danger btn-block">
-        Reject
-      </div>
+      <%= link_to 'Reject', reject_request_path(request), method: :patch, class: "card-left-bottom-button btn btn-danger btn-block" %>
       <div class="card-left-bottom-button-space"></div>
-      <div class="card-left-bottom-button btn btn-info btn-block">
-        Comment
-      </div>
+      <div class="card-left-bottom-button"></div>
       <div class="card-left-bottom-button-space"></div>
-      <div class="card-left-bottom-button btn btn-primary btn-block">
-        More
-      </div>
+      <%= link_to 'More', request, class: "card-left-bottom-button btn btn-primary btn-block" %>
```

Comment:
Yes, and it works with only instance (without request_path).
I found this in this lecture slide, and checked with the rails document.
https://karr.lewagon.org/lectures/rails/ajax-in-rails#/2/7

```
link_to "Profile", profile_path(@profile)
# => <a href="/profiles/1">Profile</a>
or the even pithier

link_to "Profile", @profile
# => <a href="/profiles/1">Profile</a>
```
http://api.rubyonrails.org/v5.1/classes/ActionView/Helpers/UrlHelper.html